nucypher/blockchain/eth/registry.py

```diff
@@ -29,7 +29,7 @@
 from twisted.logger import Logger
 
 from nucypher.blockchain.eth.networks import NetworksInventory
-from nucypher.config.constants import DEFAULT_CONFIG_ROOT
+from nucypher.config.constants import BASE_DIR, DEFAULT_CONFIG_ROOT
```

Comment:
```suggestion
from nucypher.config.constants import DEFAULT_CONFIG_ROOT
```
It seems it's not used